src/ViewInstance.js

```diff
@@ -753,13 +753,16 @@ class ViewInstance extends React.Component {
   };
 
   renderPaneTitle = (instance) => {
-    const { stripes } = this.props;
+    const {
+      stripes,
+      isShared,
+    } = this.props;
 
     return (
       <FormattedMessage
         id={`ui-inventory.${isUserInConsortiumMode(stripes) ? 'consortia.' : ''}instanceRecordTitle`}
         values={{
-          isShared: checkIfSharedInstance(stripes, instance),
+          isShared,
```

Comment:
This is a fix for the 3rd pane title only, but the `checkIfSharedInstance` condition is used in many places, and all of them can't identify the shared record, because `checkIfSharedInstance` doesn't check if the instance is shared (instance?.shared). 

I'd suggest extending `checkIfSharedInstance` and relying on it. Thus, we will have one source of truth, that can be easily corrected for all places where it is used.